### ISSUE TITLE
[iOS] BoxView with CornerRadius property set makes background color tiled in XF 3.2

### ISSUE THREAD

**Issue by brittonbeckham** [xamarin/Xamarin.Forms#3922]
### Description

When creating a BoxView using new XF 3.2 release on iOS, and if the CornerRaidus property is set, the background color is ignored and instead a "placeholder" tiled background of white and gray squares is shown. 

### Steps to Reproduce

1. Create a BoxView in iOS
2. Set the BackgroundColor or Color properties to some desired color
3. Set the CornerRadius property to something, say 50

### Expected Behavior

The BoxView should appeared filled in with the desired color, honoring the CornerRadius property

### Actual Behavior

The BoxView render space is drawn with an alternating gray / white tiled boxes instead of the desired BackgroundColor, and it is still completely square (i.e. CornerRadius is not honored).

When removing the CornerRadius property, the color returns as expected.

### Basic Information

- Version with issue: 3.2
- Last known good version: CornerRadius is a new property since 3.1 I believe, but I haven't tested on 3.1
- IDE:
- Platform Target Frameworks: 
  - iOS:  Default (12)
- Android Support Library Version: 
- Nuget Packages:
- Affected Devices:

### Screenshots

Forthcoming

**Comment by brittonbeckham** [comment]
As per something I left out in my initial description, I have the BoxView in a Grid, which is in a Frame.

``` xaml
<ViewCell xmlns="http://xamarin.com/schemas/2014/forms"
          xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
          xmlns:abstractions="clr-namespace:ImageCircle.Forms.Plugin.Abstractions;assembly=ImageCircle.Forms.Plugin"
          x:Class="Para.App.Controls.Templates.Cards.ActivityCardCellTemplate">

  <Frame HasShadow="True" Margin="15,10,15,10" Padding="20" CornerRadius="6">
    <Grid RowSpacing="0" ColumnSpacing="10">
      <Grid.ColumnDefinitions>
        <ColumnDefinition Width="Auto"/>
        <ColumnDefinition/>
      </Grid.ColumnDefinitions>
      <Grid.RowDefinitions>
        <RowDefinition Height="Auto"/>
        <RowDefinition Height="Auto"/>
        <RowDefinition Height="*"/>
        <RowDefinition Height="Auto"/>
        <RowDefinition Height="Auto"/>
        <RowDefinition Height="Auto"/>
      </Grid.RowDefinitions>

      <!--icon-->
      <BoxView WidthRequest="44" HeightRequest="44" BackgroundColor="#F0F0F0" Grid.RowSpan="2" Margin="2,2,0,0" CornerRadius="49.999" >
        <!-- NOTE: I'm doing this until the CornerRadius defect is fixed -->
        <BoxView.IsVisible>
          <OnPlatform x:TypeArguments="x:Boolean">
            <On Platform="iOS" Value="False"/>
            <On Platform="Android" Value="True"/>
          </OnPlatform>
        </BoxView.IsVisible>
      </BoxView>

      <abstractions:CircleImage BorderThickness="3" WidthRequest="45" HeightRequest="45" BorderColor="{Binding IconCircleColor}" Grid.RowSpan="2" />

      <Image Source="{Binding Icon}" Grid.RowSpan="2" WidthRequest="35" HeightRequest="35" VerticalOptions="Center" HorizontalOptions="Center"/>

      <!--title-->
      <Label Text="{Binding Title}" TextColor="{StaticResource PrimaryTextColor}"
             Grid.Column="1" FontSize="17" LineBreakMode="TailTruncation" />

      <!--category-->
      <Label Text="{Binding Category}" TextColor="{StaticResource PrimaryTextColor}"
             Grid.Column="1" Grid.Row="1" FontSize="15" />

      <!--description-->
      <Label Text="{Binding Description}" TextColor="{StaticResource PrimaryTextColor}"
             Margin="0,15,0,0"
             Grid.Row="2" Grid.ColumnSpan="2" FontSize="15" />

      <!-- recurrance -->
      <Image Source="{Binding Source='Renew', Converter={StaticResource IconResolver}}" Grid.Row="3" Grid.ColumnSpan="2" WidthRequest="16" HeightRequest="16" VerticalOptions="Center" HorizontalOptions="Start" IsVisible="{Binding ShowRecurrance}" Margin="0,1,0,0"/>
      <Label Text="{Binding RecurranceText}" TextColor="{StaticResource PrimaryTextColor}" Grid.Row="3" Grid.ColumnSpan="2" FontSize="13" IsVisible="{Binding ShowRecurrance}" VerticalOptions="Center" Margin="18,0,0,0" />

    </Grid>
  </Frame>
</ViewCell>
```

**Comment by brittonbeckham** [comment]
You can see what I am trying to do here as well (a circle with a border color and background being different). I'm plugging here that it would be nice to have a `BorderColor` and `BorderThickness` property on `BoxView` as well.

**Comment by rmarinho** [comment]
@brittonbeckham isn't this just the CornerRadius requested that is bigger than the width and height?! if i just change to something like  `CornerRadius="22" ` i have  the perfect circle i think you want. 

![screenshot_1537555188](https://user-images.githubusercontent.com/1235097/46219562-9716a600-c33f-11e8-8ecf-7745d12aeeb6.png)

**Comment by brittonbeckham** [comment]
That is a good observation. I had set the radius to 49.999 because doing 22 in Android didn't yield the full circle. Perhaps this is really an Android bug. Let me take another look at it and I'll get back.

**Comment by rmarinho** [comment]
@brittonbeckham you can be right, but that is going to be fixed with #3893 .

Closing for now reopen if it's a still a issue.
